src/java/org/apache/cassandra/tools/NodeProbe.java

```diff
@@ -1323,6 +1307,11 @@ public TabularData getFailureDetectorPhilValues()
             throw new RuntimeException(e);
         }
     }
+
+    public MessagingServiceMBean getMessagingServiceProxy()
```

Comment:
looking at this again, you could remove this since the field is set properly

Reply:
Not sure if I got it. The method is used by another command.